### PR TITLE
HAUKI-153 Show listing of possible child resources on resource page

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
image

Local link to a resource that has child resources: http://localhost:3000/resource/34144?hsa_source=tprek&hsa_username=admin%40hel.fi&hsa_created_at=2020-12-09T11%3A03%3A13.677Z&hsa_valid_until=2022-11-09T11%3A03%3A13.677Z&hsa_resource=tprek%3A8215&hsa_organization=tprek%3Adc7d39db-b35a-4612-a921-1b7b24b0baa3&hsa_signature=fe3aa3ae40903e747878c46e82933d87671a9f18b48bcbd1f9908e638e543b50

### CODE REVIEW

File: src/resource/page/ResourcePage.scss [hunk 1]
```diff
@@ -34,3 +34,11 @@
 .resource-description-text {
   white-space: pre-wrap;
 }
+
+.child-resource-description-text {
+  margin-top: 0;
+}
+
+.child-resource-list-item {
+  margin-top: 25px;
```
Comment (VilleMiekkoja) [inline src/resource/page/ResourcePage.scss:43]: - [ ] Replace this by variable from hds

**Comment by minevala** [comment]
Nice job 👍  I noticed that these subresources have pretty challenging names 😁

**Comment by VilleMiekkoja** [comment]
> Nice job 👍 I noticed that these subresources have pretty challenging names 😁

True 😁

File: src/resource/page/ResourcePage.tsx [hunk 2]
```diff
@@ -131,6 +131,9 @@ const ResourceSourceLink = ({
 
 export default function ResourcePage({ id }: { id: string }): JSX.Element {
   const [resource, setResource] = useState<Resource | undefined>(undefined);
+  const [childResources, setChildResources] = useState<Resource[] | undefined>(
```
[the remark this answers is not in the document]
Comment (VilleMiekkoja) [reply src/resource/page/ResourcePage.tsx:133]: - [ ] apply empty list as an initial value for more simplicity

File: src/resource/page/ResourcePage.tsx [hunk 3]
```diff
@@ -200,6 +210,43 @@ export default function ResourcePage({ id }: { id: string }): JSX.Element {
...
+        title="Alikohteet">
+        <p
+          data-test="child-resource-description"
+          className="resource-description-text">
+          Tässä toimipisteessä on alikohteita. Alikohteet voivat olla
+          esimerkiksi toimipisteen eri tiloja. Voit muokata alikohteiden muita
+          tietoja tilapaikkarekisterissä.
+        </p>
+        {childResources?.map((childResource, index) => {
+          return (
```
[the remark this answers is not in the document]
Comment (VilleMiekkoja) [reply src/resource/page/ResourcePage.tsx]: - [ ] Simplify by applying arrow function

**Review by minevala — APPROVED** [review]
Good job 👍

**Pull request merged**